### PR TITLE
[FIX] Ensure NLTK data

### PR DESCRIPTION
**Pull request by nikicc**
Ensure NLTK data is available at each import of `Preprocessing` module. `nltk.download` checks if it can find packages and downloads them otherwise.

### CODE REVIEW

**Comment by nikicc** [comment]
@kernc should we make this quiet — so that there are no prints on the output?

**Comment by kernc** [comment]
Nah, stderr is invisible to most people while those who have it might appreciate knowing what's going on. What we might do instead, is somehow let the also the GUI user know they now have _nltk_data_ dir in their home folder (or wherever on Windos). Perhaps with a [UserAdviceMessage](http://docs.orange.biolab.si/3/development/tutorial-utilities.html#tips) on the widget?

**Comment by nikicc** [comment]
I think it's stdout not stderr, but I agree to keep it. It's also easier to debug.

Cool, I added the UserAdviceMessage. Though, it will only be shown to those who actually open Preprocessing widget and not to others who use the default preprocessing. Still better than nothing 😄

File: orangecontrib/text/widgets/owpreprocess.py [hunk 1]
```diff
@@ -571,6 +572,11 @@ class OWPreprocess(OWWidget):
     persistent_data_filter = settings.Setting({})
     preprocessors = []  # Pre-processing modules for the current run.
 
+    UserAdviceMessages = [
+        widget.Message(
+            "NLTK data was downloaded to: {}".format(Downloader().default_download_dir()),
```
Comment (kernc) [inline orangecontrib/text/widgets/owpreprocess.py]: Could you make it just a bit more friendly? Like: "Some preprocessing methods require data (like word relationships, stop words, punctuation rules etc.) from the NLTK package. This data, if you didn't have it already, was downloaded to: {}"

Comment (nikicc) [inline orangecontrib/text/widgets/owpreprocess.py]: This sounds better 👍

**Pull request merged**